networkx/drawing/tests/test_pydot.py

```diff
@@ -28,7 +28,12 @@ def build_graph(self, G):
 
     def assert_equal(self, G1, G2):
         assert_true( sorted(G1.nodes())==sorted(G2.nodes()) )
-        assert_true( sorted(G1.edges())==sorted(G2.edges()) )
+        if G1.is_directed():
+            assert_true(sorted(G1.edges()) == sorted(G2.edges()))
+        else:
+            e1 = sorted(map(sorted, G1.edges()))
```

Comment:
Nose has assert_set_equal().  Perhaps we should build an assert_graph_equal() using the sets as you propose?  Or use the existing assert_graphs_equal() in testing/utils.py (which also sorts I see).